999-storage-import-zipped-vhds/azuredeploy.json

```diff
@@ -0,0 +1,173 @@
+            }
+        }
+    },
+    "functions": [],
+    "variables": {
+        "urlVHDZipArchive": "[parameters('Source')]",
+        "uriWritableStorageAccountBlobContainerSasToken": "[parameters('Destination')]",
+        "nestedTemplateName_ImportVHD-FromZipArchive": "[concat('ImportVHD-FromZipArchive-',uniqueString(parameters('Source'),uniqueString(parameters('Destination'))))]"
+    },
+    "resources": [
+        {
+            "type": "Microsoft.Resources/deployments",
+            "apiVersion": "2021-04-01",
+            "name": "[variables('nestedTemplateName_ImportVHD-FromZipArchive')]",
+            "properties": {
+                "mode": "Incremental",
+                "expressionEvaluationOptions": {
+                    "scope": "inner"
+                },
+                "parameters": {
+                    "location": {
+                        "value": "[parameters('location')]"
+                    },
+                    "Source": {
+                        "value": "[variables('urlVHDZipArchive')]"
+                    },
+                    "Destination": {
+                        "value": "[variables('uriWritableStorageAccountBlobContainerSasToken')]"
+                    },
+                    "deploymentScriptName": {
+                        "value": "[variables('nestedTemplateName_ImportVHD-FromZipArchive')]"
+                    }
+                },
+                "template": {
+                    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+                    "contentVersion": "1.0.0.210507",
+                    "parameters": {
+                        "location": {
+                            "type": "string",
+                            "defaultValue": "",
+                            "metadata": {
+                                "description": "The location in which the resources should be deployed."
+                            }
+                        },
+                        "Source": {
+                            "type": "string",
+                            "metadata": {
+                                "description": "URL of a ZIP archive containing VHD images files. For example: https://provider.com/bin/support/azure-vhd.zip"
+                            }
+                        },
+                        "Destination": {
+                            "type": "string",
+                            "metadata": {
+                                "description": "URI of a Storage Account container with a writable SAS Token. For example: https:// blob-storage-name.blob.core .windows.net/container-name?sp=racwl&st=2021-01-01T12:00:00&se=2021-02-01T12:00:00&spr=https&sv=2021-01-01&sig=123412341234"
```

Comment:
fix typo

Reply:
Fixed.
Same a above (it was a workaround to prevent a pipeline best practice warning)